controlplane/configparser.cpp

```diff
@@ -1670,7 +1670,20 @@ void config_parser_t::loadConfig_balancer_services(controlplane::base_t& baseNex
 			scheduler = balancer::scheduler::wlc;
 			if (exist(service_json, "scheduler_params") && exist(service_json["scheduler_params"], "wlc_power"))
 			{
-				scheduler_params.wlc_power = std::stoll(service_json["scheduler_params"]["wlc_power"].get<std::string>(), nullptr, 10);
+				auto& params = std::get<balancer::wlc_params>(scheduler_params);
+				params.wlc_power = std::stoll(service_json["scheduler_params"]["wlc_power"].get<std::string>(), nullptr, 10);
```

Comment:
we can use implicit conversion here. `nlohmann::json` has an overloaded conversion operator for `std::string`. Since `std::stoll` expects `std::string`, we can just do `service_json["scheduler_params"]["wlc_power"]`